### PR TITLE
document PR life-cycle timeouts

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

**Comment by dhellmann** [comment]
Holding this PR until the process change is approved elsewhere.

/hold

File: README.md [hunk 1]
```diff
@@ -80,3 +80,22 @@ The following labels may be applied to enhancements to help categorize them:
 - `priority/important-soon` indicates that the enhancement is related to a
 top level release priority. These will be highlighted in the
 [this-week](this-week/) newsletters.
+
+## Life-cycle
+
+
```
Comment (aravindhp) [inline README.md:86]: Nit: extra line

**Review by aravindhp — COMMENTED** [review]
/lgtm

**Pull request merged**